src/Spice86/Views/DisassemblyView.axaml

```diff
@@ -1,252 +1,235 @@
+                                        Padding="0"
+                                        Margin="0"
+                                        behaviors:InstructionPointerBehavior.IsEnabled="True">
+                            <ContentControl.Styles>
+                                <Style Selector="SelectableTextBlock">
+                                    <Setter Property="Foreground"
+                                            Value="{Binding RelativeSource={RelativeSource AncestorType=ContentControl}, Path=Foreground}" />
+                                    <Setter Property="Background"
+                                            Value="{Binding RelativeSource={RelativeSource AncestorType=ContentControl}, Path=Background}" />
+                                    <Setter Property="behaviors:UseParentListBoxContextMenuBehavior.UseParentContextMenu" Value="True" />
+                                </Style>
+                            </ContentControl.Styles>
+
+                            <Grid ColumnDefinitions="Auto, Auto, *,4*,2*" Margin="0">
+                                <!-- Breakpoint indicator -->
+                                <Border Grid.Column="0"
+                                        Width="16"
+                                        Height="16"
+                                        Background="Transparent"
+                                        Tapped="OnBreakpointClicked"
+                                        Cursor="Hand">
+                                    <Ellipse
+                                        Width="12"
+                                        Height="12"
+                                        HorizontalAlignment="Center"
+                                        VerticalAlignment="Center"
+                                        Fill="{Binding Breakpoint.IsEnabled, FallbackValue=Transparent, Converter={StaticResource BreakpointColorConverter}}"
+                                        Margin="0,0,4,0" />
+                                </Border>
+
+                                <!-- Address -->
+                                <SelectableTextBlock Grid.Column="1"
+                                                     Text="{Binding SegmentedAddress}"
+                                                     Foreground="#78C9B0"
+                                                     FontWeight="SemiBold"
+                                                     Margin="0,0,6,0" />
+
+                                <!-- Bytes -->
+                                <SelectableTextBlock Grid.Column="2"
+                                                     Text="{Binding ByteString}"
+                                                     Foreground="#B5CEA8"
+                                                     FontFamily="Consolas,Menlo,Monaco,monospace"
+                                                     FontSize="11"
+                                                     Width="110"
```

Comment:
Use MinWidth and MinHeight instead of fixed Width and Height

Reply:
Done.